server/immich-openapi-specs.json

```diff
@@ -4357,16 +4357,6 @@
       "put": {
         "operationId": "updateConfig",
         "parameters": [],
-        "requestBody": {
-          "content": {
-            "application/json": {
-              "schema": {
-                "$ref": "#/components/schemas/SystemConfigDto"
```

Comment:
Something is weird is going on here, `SystemConfigDto` get removed from the body, which lead to failed test on the web

Reply:
@jrasm91 Could this be related to this #4403?

Reply:
> @jrasm91 Could this be related to this #4403?

I tried to revert that commit from `main` and the spec file is now correct. I wonder what would cause it 🤔 ?

Reply:
I can take a look later. In this PR though, just don't commit the removal of that section or the associated client changes.